newdle/client/src/client.js

```diff
@@ -1,14 +1,12 @@
 import flask from 'flask-urls.macro';
 import {useReducer} from 'react';
 import {getToken, isAcquiringToken} from './selectors';
-import {tokenExpired, tokenNeeded} from './actions';
+import {tokenExpired, tokenNeeded, addError} from './actions';
 
 function backendReducer(state, action) {
   switch (action.type) {
     case 'submit':
       return {...state, submitting: true, error: '', result: null};
-    case 'error':
```

Comment:
I think we should keep this; a component may want to know that a call it did failed